packages/web/localizations/ko.json

```
@@ -289,7 +289,7 @@
       "poweredBy": "제공:",
       "proceed": "진행하다",
       "depositExpiration": "이 입금 주소는 다음 날짜에 만료됩니다.",
-      "undergoAudit": "감사 중(2024년 1월 완료 예정)",
+      "undergoAudit": "오딧 진행중",
```

```suggestion
      "undergoAudit": "감사 진행중",
```
I believe that "감사" deliver its meaning more accuratly.